client/app/scripts/services/repos.js

```diff
@@ -48,7 +48,7 @@ angular.module('clientApp')
      */
     function getDataFromBackend(repoId) {
       var deferred = $q.defer();
-      var url = Config.backend + '/api/github_repos';
```

Comment:
this cannot change - it what gets the user repos from github. you can create another service.